tensorboard/plugins/graphs/tf_graph_loader/tf-graph-loader.html

```diff
@@ -61,6 +61,7 @@
       readOnly: true,
       notify: true
     },
+    /** @type {Object?} */
```

Comment:
They probably did that because `Object` can mean a vast universe of types, that Polymer definitions aren't able to express, yet the Closure Compiler can. So Closure Compiler probably thinks, "ok they almost certainly want a type annotation here, but I'll forgive them if they put a comment saying Object is what they really want."